extension/encoding/textencodingextension/README.md

```diff
@@ -9,4 +9,20 @@
 | [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | [@MovieStoreGuy](https://www.github.com/MovieStoreGuy), [@atoulme](https://www.github.com/atoulme) |
 
 [development]: https://github.com/open-telemetry/opentelemetry-collector#development
-<!-- end autogenerated section -->
\ No newline at end of file
+<!-- end autogenerated section -->
+
+The `text_encoding` extension is an encoding extension that can unmarshal and marshal logs.
+
+The extension accepts an encoding and separator to unmarshal data as the body of one or more log records.
+The separator accepts regular expressions.
+
+When marshaling logs, the extension will return the body content, separated by a separator.
```

Comment:
I think the docs on the marshaling separator and unmarshaling separator could be made a bit clearer, perhaps something that can be amended in a follow-up pull request.